nltk/tokenize/texttiling.py

```diff
@@ -256,18 +259,16 @@ def _create_token_table(self, token_sequences, par_breaks):
                     if token_table[word].last_tok_seq != current_tok_seq:
                         token_table[word].last_tok_seq = current_tok_seq
                         token_table[word]\
-                                .ts_occurences.append([current_tok_seq,1])
+                            .ts_occurences.append([current_tok_seq, 1])
```

Comment:
In this situation, I think it's fine to overwrite PEP8 and go beyond the 80 chars margin, put it in one line. 

```
token_table[word].ts_occurences.append([current_tok_seq, 1])
```